src/sketchup_objects/spring_link.rb

```diff
@@ -23,6 +23,7 @@ def initialize(first_node, second_node, edge, spring_parameters: nil, id: nil)
   def spring_parameters=(parameters)
```

Comment:
Rubocop has a point here, should be ```:attr_reader spring_parameters``` in line 8

Reply:
Done in de186d8